### ISSUE TITLE
Module not working for other Sbert models

### ISSUE THREAD

**Issue by nickmuchi87** [Pandora-Intelligence/fast-sentence-transformers#1]
Hi there, I tried running the fast transformer on all-mpnet-base-v2 and also multi-qa-mpnet-base-cos-v1 and got an error:

Encoder=SentenceTransformer("all-mpnet-base-v2",device='cpi',quantize=True)

TypeError:

Forward() takes from 1 to 9 positional arguments but 10 were given.

**Comment by davidberenstein1957** [comment]
@nickmuchi87 I understand the issue, and will take a look over the course of the coming weeks, since I only focused on the default embedding models.

**Comment by loretoparisi** [comment]
@davidberenstein1957 any news about support to `all-mpnet-base-v2`? 
It seems default onnx converter [fails](https://github.com/UKPLab/sentence-transformers/issues/1283), I get this error in fact:

```
python -m transformers.onnx --model=sentence-transformers/all-mpnet-base-v2 onnx/
Downloading tokenizer_config.json: 100%|████████████████████████████████████████████████████████| 363/363 [00:00<00:00, 244kB/s]
Downloading config.json: 100%|██████████████████████████████████████████████████████████████████| 571/571 [00:00<00:00, 471kB/s]
Downloading vocab.txt: 100%|█████████████████████████████████████████████████████████████████| 226k/226k [00:00<00:00, 39.8MB/s]
Downloading tokenizer.json: 100%|████████████████████████████████████████████████████████████| 455k/455k [00:00<00:00, 50.1MB/s]
Downloading special_tokens_map.json: 100%|██████████████████████████████████████████████████████| 239/239 [00:00<00:00, 212kB/s]
Downloading pytorch_model.bin: 100%|█████████████████████████████████████████████████████████| 418M/418M [00:05<00:00, 74.1MB/s]
Traceback (most recent call last):
  File "/anaconda3/envs/JupyterSystemEnv/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/anaconda3/envs/JupyterSystemEnv/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/anaconda3/envs/JupyterSystemEnv/lib/python3.7/site-packages/transformers/onnx/__main__.py", line 107, in<module>
    main()
  File "/anaconda3/envs/JupyterSystemEnv/lib/python3.7/site-packages/transformers/onnx/__main__.py", line 76, in main
    model_kind, model_onnx_config = FeaturesManager.check_supported_model_or_raise(model, feature=args.feature)
  File "/anaconda3/envs/JupyterSystemEnv/lib/python3.7/site-packages/transformers/onnx/features.py", line 584, incheck_supported_model_or_raise
    model_features = FeaturesManager.get_supported_features_for_model_type(model_type, model_name=model_name)
  File "r/anaconda3/envs/JupyterSystemEnv/lib/python3.7/site-packages/transformers/onnx/features.py", line 487, inget_supported_features_for_model_type
    f"{model_type_and_model_name} is not supported yet. "
KeyError: "mpnet is not supported yet. Only ['albert', 'bart', 'beit', 'bert', 'big-bird', 'bigbird-pegasus', 'blenderbot', 'blenderbot-small', 'bloom', 'camembert', 'codegen', 'convbert', 'convnext', 'data2vec-text', 'deberta', 'deberta-v2', 'deit', 'detr', 'distilbert', 'electra', 'flaubert', 'gpt2', 'gptj', 'gpt-neo', 'ibert', 'layoutlm', 'layoutlmv3', 'levit', 'longt5', 'marian', 'mbart', 'mobilebert', 'mobilevit', 'm2m-100', 'perceiver', 'resnet', 'roberta', 'roformer', 'squeezebert', 't5', 'vit', 'xlm', 'xlm-roberta', 'yolos'] are supported. If you want to support mpnet please propose a PR or open up an issue."
```

**Comment by nickmuchi87** [comment]
Any news on this one?

**Comment by davidberenstein1957** [comment]
Sadly, I have not gotten around to looking into this. Also, the ONNX conversion error stems from the original `transformers` project. They are working on it here: https://github.com/huggingface/transformers/pull/19009).

**Comment by davidberenstein1957** [comment]
Look into more direct integration with `transformers`

https://huggingface.co/docs/transformers/serialization 
xlm model achitectures are supported by huggingface
POC
```bash
python -m transformers.onnx --model="microsoft/Multilingual-MiniLM-L12-H384" onnx/                                                                                                          
```

```python
from transformers import AutoTokenizer
from onnxruntime import InferenceSession

tokenizer = AutoTokenizer.from_pretrained("microsoft/Multilingual-MiniLM-L12-H384")
session = InferenceSession("onnx/model.onnx")
# ONNX Runtime expects NumPy arrays as input
inputs = tokenizer("Using DistilBERT with ONNX Runtime!", return_tensors="np")
outputs = session.run(output_names=["last_hidden_state"], input_feed=dict(inputs))
print(outputs)

```

**Comment by davidberenstein1957** [comment]
closed in favor of #5
